src/test/kotlin/no/nav/yrkesskade/saksbehandling/repository/BehandlingRepositoryTest.kt

```diff
@@ -36,12 +36,26 @@ class BehandlingRepositoryTest : AbstractTest() {
     fun `hent egne behandlinger`() {
         val sak = sakRepository.findAll().first()
         genererBehandling(2L, "test", Behandlingsstatus.IKKE_PAABEGYNT, sak)
+        genererBehandling(3L, "test", Behandlingsstatus.FERDIG, sak)
         val behandling = genererBehandling(1L, "test", Behandlingsstatus.UNDER_BEHANDLING, sak)
 
         behandlingRepository.save(behandling)
 
-        val behandlinger = behandlingRepository.findBySaksbehandlingsansvarligIdent("test", Pageable.ofSize(10))
-        assertThat(behandlinger.size).isEqualTo(1)
+        val behandlinger = behandlingRepository.findBySaksbehandlingsansvarligIdentAndStatus("test", Behandlingsstatus.UNDER_BEHANDLING, Pageable.ofSize(10))
+        assertThat(behandlinger.numberOfElements).isEqualTo(1)
```

Comment:
Kanskje en sjekk på at resultatet har riktig behandlingsstatus og ikke bare antall? Samme for testene under